runtime/lua/vim/shared.lua

```diff
@@ -174,6 +174,21 @@ function vim.tbl_filter(func, t)
   return rettab
 end
 
+--- Create a table with a given value repeated multiple times.
+---
+--- Does *not* copy the provided object, so the returned table contains
+--- identical objects.
+---
+--@param value the value to repeat
+--@param count integer the number of times to repeat the given value
```

Comment:
I think we use `---@` in here ? I never remember those...

Reply:
I used the style from the surrounding code (e.g. the function right below) but I think I caught out of the corner of my eye that that's being changed? But yeah e.g. https://github.com/neovim/neovim/blob/master/runtime/lua/vim/shared.lua#L59-L64 uses I believe what I did so I kept it the same for now unless you want me to use the new style just for this?

Reply:
Probably best to match local style and do a batch refactor when someone gets bored.

Reply:
> Probably best to match local style and do a batch refactor when someone gets bored.

That someone is me :) https://github.com/neovim/neovim/pull/15255

I'd suggest just using `---@` and saving the trouble of having to do another docstring update down the road.

Reply:
OK, done.